pybadge-high/Cargo.toml

```diff
@@ -33,9 +33,9 @@ time = []
 
 [dependencies]
 cortex-m = "0.7.7"
-embedded-graphics = "0.7.1"
-edgebadge = { version = "0.9.0"}
-st7735-lcd = { version = "0.8.1", default-features = false }
+embedded-graphics = "0.8.1"
+edgebadge = { version = "0.9.0", git = "https://github.com/LuckyTurtleDev/atsamd.git", branch = "update-embedded-graphics"}
```

Comment:
Is it a good idea to use a fork just to have a transitive dependency updated? Maybe we should just wait until edgebadge merges and releases the PR.

Reply:
Yes of course. It is for testing only.